docs/tutorials/bids_archive_tutorial.ipynb

```diff
@@ -0,0 +1,755 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The last piece of data we'll see how to get from an archive is the events file corresponding to a particular scanning run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Event files to get can be filtered by entities, as with \n",
+    "# getImages and getSidecarMetadata\n",
+    "events = archive.getEvents(subject='01', \n",
+    "                           task='languageproduction', run=1)\n",
+    "\n",
+    "# All event files can be retrieved when specifiying no entities\n",
+    "events = archive.getEvents()\n",
+    "\n",
+    "# Event files are returned as BIDSDataFile objects\n",
+    "# See the PyBids documentation for more information on those\n",
+    "eventsFile = events[0]\n",
+    "print('Events file type: ', type(eventsFile))\n",
+    "\n",
+    "# One method of the BIDSDataFile object returns\n",
+    "# a Pandas data frame of the events file\n",
+    "eventsDF = eventsFile.get_df()\n",
+    "\n",
+    "print(\"Sample data: \\n\", eventsDF[:][:5])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# BIDS Archive: Getting & Appending Incrementals\n",
+    "\n",
+    "One of the most important functions that a BIDS Archive enables in the context of RT-Cloud is working with `BIDS Incrementals`. From a `BIDS Archive`, you can get image data and metadata and package it into a `BIDS Incremental` using `getIncremental`. The opposite operation, for when you already have a `BIDS Incremental` and want to either append it to an existing archive (or create a new archive), is to append it to a `BIDS Archive` using `appendIncremental`.\n",
+    "\n",
+    "For example, if you have a complete dataset that you want to test a new real-time experiment on, you can using `getIncremental` repeatedly to iterate over your entire dataset, streaming the resulting BIDS Incrementals to RT-Cloud and the new experimental script you want to try out. Then, when you're running your new experiment in RT-Cloud for real, as `BIDS Incremental` files are streamed from the scanner to your script, you can create a new `BIDS Archive` in an empty folder on the computer running your script, and build up the archive of your entire experiment by calling `appendIncremental` for each `BIDS Incremental` you receive. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Setup\n",
+    "td = tempfile.TemporaryDirectory()\n",
+    "print('Temporary directory path:', td.name)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set up an archive and get an incremental\n",
+    "firstIncremental = archive.getIncremental(subject='01', task='languageproduction')\n",
```

Comment:
firstIncremental is a bit confusing, why are we getting an incremental with all the sub-01 volumes?

Reply:
The default `imageIndex` parameter is 0, so by default it'll extract the first image of the sub-01 languageproduction volume. I've added that parameter in explicitly now for clarity (and renamed it to imageIndex instead of sliceIndex to complete the slice -> volume transition)